File: CS/Controllers/EFDatabaseServerModeController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using EF.Models;

namespace EF.Controllers {
    public class EFDatabaseServerModeController : Controller {
        NorthwindContext db = new NorthwindContext();

        public ActionResult Index() {
            return View();
        }

        public ActionResult GridViewPartial() {
            return PartialView(db.Orders);
        }
    }
}

File: VB/Controllers/EFDatabaseServerModeController.vb
Imports System.Web.Mvc
Imports EF.Models

Namespace EF.Controllers

    Public Class EFDatabaseServerModeController
        Inherits Controller

        Private db As NorthwindContext = New NorthwindContext()

        Public Function Index() As ActionResult
            Return View()
        End Function

        Public Function GridViewPartial() As ActionResult
            Return PartialView(db.Orders)
        End Function
    End Class
End Namespace
